package.json

```diff
@@ -74,10 +74,10 @@
     "@pnpm/tabtab": "0.5.4",
     "ansi-escapes": "7.0.0",
     "ansi-to-html": "0.7.2",
+    "ansis": "^3.17.0",
```

Comment:
@serhalp,
meanwhile I have released a new version - 4.0.0. 
This release brings a stable and more compact ANSI library.
v4 is ~15.7% smaller than v3.17.